templates/settings.yaml.erb

```diff
@@ -70,3 +70,10 @@
 
 :dynflow:
   :pool_size: <%= scope.lookupvar("foreman::dynflow_pool_size") %>
+<% if scope.lookupvar("foreman::apache") && !scope.lookupvar("foreman::passenger") -%>
+
+# Configure reverse proxy headers
+:ssl_client_dn_env: HTTP_SSL_CLIENT_S_DN
+:ssl_client_verify_env: HTTP_SSL_CLIENT_VERIFY
+:ssl_client_cert_env: HTTP_SSL_CLIENT_CERT
+<% end -%>
```

Comment:
Based on our discussions, I thought you were gonna keep this still being set in the Settings (database) vs moving to SETTINGS (app restart). Don't get me wrong I like this change given what these represent, but I'd do this stand alone so we can make the code change for moving them alongside it easier.

Reply:
At this point I first want to get this working, then I'll see whether we use the config file or database. First I want to get the Apache config right.

Reply:
Right... my point is the application won't respect these because they are SETTINGS and it currently is built to read Settings (database).

I was wondering if for now we'd be best doing:

```
    foreman_config_entry { 'ssl_client_cert_env':
      value => 'HTTP_SSL_CLIENT_CERt',
    }
```

Reply:
Settings that are in settings.yaml are visible, but can't be overridden:

![screen shot 2018-11-14 at 19 33 11](https://user-images.githubusercontent.com/155810/48504110-418d4e80-e844-11e8-8d1d-284279f528e5.png)

Note it's visible, but there's no edit button. The's also a tooltip:
> This setting is defined in the configuration file 'settings.yaml' and is read-only.

Reply:
Huh, never noticed that before.